test/windows_zones.jl

```diff
@@ -0,0 +1,3 @@
+using TimeZones: WINDOWS_TRANSLATION
+
+@test WINDOWS_TRANSLATION["Central European Standard Time"] == "Europe/Warsaw"
```

Comment:
```suggestion
@test occursin(r"^release-\d+(-\d+)$", UNICODE_CLDR_VERSION)
@test WINDOWS_TRANSLATION["Central European Standard Time"] == "Europe/Warsaw"
```

Reply:
Thanks, I needed to add the `?` after the last group since a second number is optional; `release-43-1` and `release-45` are both valid.

Reply:
My bad. I added the grouping to make it optional but just forgot to add the `?` modifier